src/routes/tokens/ApiTokens.tsx

```diff
@@ -22,7 +25,8 @@ import GenerateTokenModal from './components/GenerateTokenModal';
 const ApiTokens = (): JSX.Element => {
     const [activeTokens, setActiveTokens] = useState<BearerTokenT[]>([]);
     const [generatedToken, setGeneratedToken] = useState<NewBearerTokenT>();
-
+    const key = useKeyFromPath(PATHS.MANAGE_TOKENS);
+    useAssetListDocumentTitleEffect('Tokens management', key);
```

Comment:
```suggestion
    useDocumentTitleEffect(
        (setDocumentTitle) => setDocumentTitle('Tokens management'),
        []
    );
```
is all you need as there's no drawer for tokens